.github/workflows/test.yml

```diff
@@ -81,7 +81,7 @@ jobs:
       - name: Run multiplexer tests
         run: make test-multiplexer
 
-  test-docker-e2e:
+  test-e2e:
```

Comment:
perhaps could have been separated into a different PR, but as @rootulp pointed out, we can remove old workflows with test-e2e, so I figured we could also replace test-docker-e2e since we will now have a single set of e2e tests